vector/src/main/java/im/vector/adapters/ParticipantAdapterItem.java

```diff
@@ -433,9 +432,8 @@ public void displayAvatar(MXSession session, ImageView imageView) {
         if (null != getAvatarBitmap()) {
             imageView.setImageBitmap(getAvatarBitmap());
         } else {
-            if ((null != mUserId) && (android.util.Patterns.EMAIL_ADDRESS.matcher(mUserId).matches()) || !mIsValid) {
-                imageView.setImageBitmap(VectorUtils.getAvatar(imageView.getContext(), VectorUtils.getAvatarColor(mIsValid ? mUserId : ""), "@@", true));
-            } else {
+            //No@ avatar on tchap
```

Comment:
Quel sera le comportement attendu en absence d'image et d'avatar ? Un placeholder est-il prévu ?

Reply:
In that case it's the first letter of displayname

Reply:
ok